@here/harp-mapview-decoder/lib/WorkerService.ts

```diff
@@ -7,7 +7,7 @@
 import { WorkerServiceProtocol } from "@here/harp-datasource-protocol";
 import { LoggerManager } from "@here/harp-utils";
 
-const logger = LoggerManager.instance.create("WorkerService", { enabled: false });
+const logger = LoggerManager.instance.create("WorkerService", { enabled: true });
```

Comment:
I was getting some issues in the decoder, but it wasn't logging, so I have enabled the logging.